libs/sdk-ui-kit/src/utils/immutable.ts

```diff
@@ -0,0 +1,20 @@
+// (C) 2021 GoodData Corporation
+
+import isEqualWith from "lodash/isEqualWith";
+import { isImmutable } from "immutable";
+
+/**
+ * Determine if 'props' and 'nextProps' objects are equal
+ *
+ * @param props
+ * @param nextProps
+ * @returns true if yes, false if they are not equal
+ *
+ * @internal
+ */
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function propsEqual(props: any, nextProps: any): boolean {
```

Comment:
Nitpick: You could probably avoid this eslint warning by making the function generic on both object types.

Reply:
That is a really good suggestion! Fixed.